lib/pages/home_page.dart

```diff
@@ -1,59 +1,74 @@
 import 'package:flutter/material.dart';
 
 class HomePage extends StatefulWidget {
-  const HomePage({super.key});
+  const HomePage({Key? key}) : super(key: key);
 
   @override
   _HomePageState createState() => _HomePageState();
 }
 
 class _HomePageState extends State<HomePage> {
-  List<String> tasks = [
-    'Task 1',
-    'Task 2',
-    'Task 3',
-    'Task 4',
-  ];
-
+  List<String> tasks = ['Task 1', 'Task 2', 'Task 3', 'Task 4'];
```

Comment:
Maybe better to create some object for this entity)?
It's not really comfortable to manage 2 list